project/plugins.sbt

```diff
@@ -17,7 +17,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.3")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-s3" % "0.5")
 
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.12")
```

Comment:
@bantonsson just released sbt-jmh `0.1.13` with updated JMH version to `1.7.1` (the 1.12 shipped with 1.6.x).

(after holidays of course :-))